nvflare/app_opt/xgboost/histogram_based_v2/secure_data_loader.py

```diff
@@ -16,9 +16,12 @@
 
 from nvflare.app_opt.xgboost.data_loader import XGBDataLoader
 
+COL_SECURE = 2
+ROW_SECURE = 3
+
 
 class SecureDataLoader(XGBDataLoader):
-    def __init__(self, rank: int, folder: str):
+    def __init__(self, rank: int, folder: str, data_split_mode=COL_SECURE):
```

Comment:
can you add to docstring to explain this? thanks!

Reply:
Will add in next PR.